app/components/contacts/AllContacts.js

```diff
@@ -69,7 +70,7 @@ class AllContacts extends Component<Props, State> {
         </SearchRow>
         <ListsWrapper isModalMode={isModalMode}>
           <div>
-            {lastUsedAddresses && lastUsedAddresses.length && (
+            {lastUsedAddresses && lastUsedAddresses.length ? (
```

Comment:
latestUsedAddresses are composed from tx objects and not Contact, this will cause ContactsList to fail

Reply:
mapped to include only contact props.